src/components/common/input/label.tsx

```diff
@@ -0,0 +1,16 @@
+import {LabelHTMLAttributes} from "react";
+import style from "./input.module.scss";
+interface labelProps extends LabelHTMLAttributes<HTMLInputElement> {
+  text: string;
+}
+function Label({htmlFor, text}: labelProps) {
+  return (
+    <label htmlFor={htmlFor} className={style.label}>
+      {text}
```

Comment:
여기도 children으로 표현되는게 더 좋을 것 같아요!
```suggestion
      {children}
```